starboard/nplb/player_test_util.cc

```diff
@@ -39,6 +39,22 @@ using std::placeholders::_3;
 using std::placeholders::_4;
 using testing::FakeGraphicsContextProvider;
 
+const char* kAudioTestFiles[] = {"beneath_the_canopy_aac_stereo.dmp",
+                                 "beneath_the_canopy_opus_stereo.dmp"};
+// "sintel_329_ec3.dmp", "sintel_381_ac3.dmp"};
```

Comment:
Can you provide more info on why we are removing them?

Reply:
recovered.